doc/connectivity/bluetooth/overview.rst

```diff
@@ -58,11 +59,17 @@ Bluetooth stack.
 
     * Peripheral & Central
     * Observer & Broadcaster
+    * Multiple PHY support (2Mbit/s, Coded)
+    * Extended Advertising
+    * Periodic Advertising (including Sync Transfer)
```

Comment:
The Zephyr controller does not support sync transfer yet

Reply:
Very true, oversight on my part.

Reply:
That's in the Host section